### DISCUSSION TITLE
Wiki not showing?

### DISCUSSION THREAD

**Discussion by handdara** [nvim-telekasten/telekasten.nvim#318] [Q&A] [answered]
It looks like there should be a wiki based on the main readme which mentions/links to it. However, i can't see it, is this a me issue or has it been disabled?

Love the plugin btw

**Comment by lambtho12** [comment, marked as answer]
Sorry for the confusion. I just removed the mention of the wiki from the README (39c7945d6d209ab921c0ea2b9328d5627e66a6f4).

The wiki had not been maintained for a long time (we barely have enough time to maintain the code itself to be honest). It was causing confusion as users were following outdated advice, etc. It was removed a few weeks ago after discussion in issue #307.

> Love the plugin btw

Thank you very much !
